keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterName.kt

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+/**
+ * Frigga style name.
+ */
+data class ClusterName(
```

Comment:
What's the relationship between this and Moniker?

Reply:
Pretty much the same thing. I'm still thinking this out. I'm not sure if it's better for the user to specify the cluster name using separate values like this or for us to parse it out of a single name string.